### ISSUE TITLE
[4.1] Article Modal

### ISSUE THREAD

**Issue by heelc29** [joomla/joomla-cms#37841]
### Steps to reproduce the issue

Open article modal in front- and backend
![image](https://user-images.githubusercontent.com/66922325/169630300-da9cc698-b071-4d21-879a-0202ab92e522.png)
![image](https://user-images.githubusercontent.com/66922325/169630456-15b81e0d-6e8a-4ab0-9094-28ac4fe9cda9.png)

### Expected result

- Ordering field in Frontend Modal should match with list
- Front- and Backend filter fields should be identical
- Filter fields in frontend do look the same

### Actual result

- Ordering field in Frontend Modal do not match with list
- Front- and Backend filter fields are different
- Filter fields in frontend do not look the same

**Comment by chmst** [comment]
@heelc29 This is intended behavoiur. 
Users in frontend don't work with IDs but are more familiar with alphabetical ordering. 
Admins in Backend use to work on new articles and it saves time when articles are ordered from newest to latest. 

Do you think this is wrong?

**Comment by heelc29** [comment]
@chmst I don't know, what the better default sorting is. But at the moment the ordering field and the real ordering do not match:
![image](https://user-images.githubusercontent.com/66922325/169781485-8be2a2c9-d0a7-40df-b802-0698faf0a3fd.png)

**Comment by heelc29** [comment]
@Mayankq visit your.domain/index.php?option=com_content&view=form&layout=edit
And open the content editor-xtd:
![image](https://user-images.githubusercontent.com/66922325/221360422-921edb08-e4ce-4261-ba96-7d544f74ded0.png)
![image](https://user-images.githubusercontent.com/66922325/221360435-1363e144-8655-4949-bb2e-1b455f4fcb4d.png)

**Comment by Mohammadkhan18** [comment]
yeah i got the issue but for making the changes should we just clone the joomla file or have run the joomla on localhost

**Comment by Programmer-Dost** [comment]
Can someone help me with reproducing this issue? as the steps to open Article Modal are unclear.

**Comment by chmst** [comment]
If you add a new Menu Item and select Article / single article as menu Item Type, you have to select an article. 
This opens a modal window with articles. 
The modal window enables the user the selection - but it has not the same columns as the Article overview.

**Comment by alikon** [comment]
i was unable to reproduce
are still someone able to reproduce it and give us  good info on how to reproduce ?

**Comment by brianteeman** [comment]
@heelc29 and @chmst have both provded instructions how to replicate

**Comment by alikon** [comment]
i've followed that , that's why i'm asking if it is still reproducible, i was not able to...

**Comment by brianteeman** [comment]
yes its still reproducable

**Comment by TanyaGupta1901** [comment]
Is this fixed?

**Comment by TheLostShip** [comment]
I'm a bit confused folks. I mean, where am I gonna find this <strong>Article</strong> feature on the <a href="joomla.org" target="_blank">website</a> so that I may check the issue myself in order to try fixing it? It's going to be my first contribution, can anyone help me out?

**Comment by charvimehradu** [comment]
@TheLostShip You'll need to set up your Joomla environment to work with it. You can find the related documentation [here](https://docs.joomla.org/J4.x:Setting_Up_Your_Local_Environment).

**Comment by TheLostShip** [comment]
> @TheLostShip You'll need to set up your Joomla environment to work with it. You can find the related documentation [here](https://docs.joomla.org/J4.x:Setting_Up_Your_Local_Environment).

Ah, I see. Thanks.

**Comment by KaranNegi20Feb** [comment]
hey @heelc29, is this issue still available to be solved?

**Comment by Payoshnik123** [comment]
@heelc29 could you please assign me this isssue?

**Comment by fgsw** [comment]
@Payoshnik123 Issues aren't assigned. If you wanna create a pull request, go for it.

**Comment by Dileepadari** [comment]
Hi,
Could you clarify whether this issue is about changing the default label or the default option?

If it's a label change, I've already changed it and attached the PR below. Could someone confirm if this aligns with the expected change?
PR: [https://github.com/joomla/joomla-cms/pull/45106](https://github.com/joomla/joomla-cms/pull/45106)

**Comment by richard67** [comment]
>- Ordering field in Frontend Modal do not match with list
>- Front- and Backend filter fields are different
>- Filter fields in frontend do not look the same

@heelc29 As far as I can see, @Dileepadari 's PR #45106 fixes only the first issue. Are the other 2 meanwhile clarified, or do you still think they should be fixed? If clarified, then the PR would fix this issue completely, but if not, it would fix it only partly and we should leave this issue here open.

**Comment by Dileepadari** [comment]
Oh, Thanks Richard for the clarification, I think they haven't clarified regarding the other two issues. But I think its better to have same styling across the frontend and backend. I will look into the other two issues too.

**Comment by Dileepadari** [comment]
@richard67 or @heelc29 , could you please clarify the following?  

For the second issue: **"Front- and Backend filter fields are different"**  
- Is the problem related to the **missing "Featured" filter**, or is it about differences in their **grid division/layout**?  

For the third issue: **"Filter fields in frontend do not look the same"**  
- Does this mean that the **spacing between the filters** is inconsistent, or is there another visual discrepancy?  

Looking forward to your insights. Thanks!

**Comment by Rohit-rk07** [comment]
i see a complete different interface in frontend and backend. Could u clarify if the issue is still open or not.

**Comment by brianteeman** [comment]
> i see a complete different interface in frontend and backend. Could u clarify if the issue is still open or not.

then you are looking in the wrong place or doing something else that is wrong. impossible to guess what you are doing wrong as I dont know what you are seeing

**Comment by Rohit-rk07** [comment]
![Image](https://github.com/user-attachments/assets/2eaf861a-4522-46e2-b58b-6dbcc07d2507)

![Image](https://github.com/user-attachments/assets/16a163c4-efdb-4e93-a689-4f1ba5c30ba3) 
Have a look this is what iam seeing and in backend ordering seems to be okay

**Comment by fgsw** [comment]
> Open article modal in front- and backend

@Rohit-rk07 You have to open the article **modal**.

**Comment by Rohit-rk07** [comment]
would u please guide how ?

**Comment by heelc29** [comment]
> For the second issue: **"Front- and Backend filter fields are different"**
> 
> * Is the problem related to the **missing "Featured" filter**, or is it about differences in their **grid division/layout**?

Its the missing "Featured" filter and the order of filters

> For the third issue: **"Filter fields in frontend do not look the same"**
> 
> * Does this mean that the **spacing between the filters** is inconsistent, or is there another visual discrepancy?

its the background of the arrow button

![Image](https://github.com/user-attachments/assets/214379b8-63af-4087-9859-c48a1fc0c4ef)

**Comment by Rohit-rk07** [comment]
@heelc29 Thanks for clarifying the issue.  Does this #45189 solves the first two issue.
And about the third one, i can see the bg of arrow buttons are same in all filter fields. So I guess it should be okay.

![Image](https://github.com/user-attachments/assets/cf1cf3ed-b7e0-4284-bc85-1b922ce77838)

**Comment by KamalDeveloper11** [comment]
Hi, I would like to work on this issue.  
My contribution will focus on the Frontend part (HTML/CSS/JS).  
Please assign it to me if possible.

**Comment by SRV-KILLER09** [comment]
Hi everyone! 👋
I’d like to work on the frontend part of this issue (#37841), focusing on aligning the filter fields and making the dropdown styling consistent with the backend. 

I’ll first verify the current state locally and then submit a PR with any necessary fixes.

Thanks! 🙂

**Comment by Serenehue** [comment]
Hi! I’m planning to work on a small frontend/CSS fix for this issue.
I’ll submit a PR soon. Thanks!

**Comment by shaily-ux** [comment]
> ### Steps to reproduce the issue
> Open article modal in front- and backend ![image](https://user-images.githubusercontent.com/66922325/169630300-da9cc698-b071-4d21-879a-0202ab92e522.png) ![image](https://user-images.githubusercontent.com/66922325/169630456-15b81e0d-6e8a-4ab0-9094-28ac4fe9cda9.png)
> 
> ### Expected result
> * Ordering field in Frontend Modal should match with list
> * Front- and Backend filter fields should be identical
> * Filter fields in frontend do look the same
> 
> ### Actual result
> * Ordering field in Frontend Modal do not match with list
> * Front- and Backend filter fields are different
> * Filter fields in frontend do not look the same



Hi, I am new to Joomla and I am trying to understand this issue.
Could you please confirm if the problem is related to the article modal filtering/sorting behavior?

**Comment by shubhamftp01** [comment]
I am new to Joomla and would like to work on this issue. Can you guide me on where to start?
